code/controllers/subsystem/SSair.dm

```diff
@@ -16,87 +15,65 @@ SUBSYSTEM_DEF(air)
 	runlevels = RUNLEVEL_GAME | RUNLEVEL_POSTGAME
 	offline_implications = "Turfs will no longer process atmos, and all atmospheric machines (including cryotubes) will no longer function. Shuttle call recommended."
 	cpu_display = SS_CPUDISPLAY_HIGH
```

Comment:
Will this still be the case?

Reply:
Yup, because the subsystem is what tells Rust to run atmos ticks. If the subsystem is offline, atmos isn't going to flow.

Reply:
I meant that this would show in the `MC|H` tab, marking it as a high CPU SS, as something that could easily bog the server. The var has nothing to do with cpu allocation stuffs.